Script/olterra.dat

```diff
@@ -210,6 +211,7 @@ decoration
     BitmapPos = 16, 368;
     HPModifier = 25;/*
     IsAlwaysTransparent = false;*/
+    LeftOverItems == banana { Times = 0:1; }
```

Comment:
Well, bananas grow on a (specific type of) palm, don't they? But green bananas are nice idea, ripening instead of rotting. Maybe again with only a small chance to drop from any given palm?

But now that I'm looking at it, there are quite a few palms in NA, so maybe I should revert this change?